resources/web/wwi/x3d.js

```diff
@@ -1073,3 +1075,17 @@ function convertStringTorgb(s) {
   var v = convertStringToVec3(s);
   return new THREE.Color(v.x, v.y, v.z);
 }
+
+function horizontalToVerticalFieldOfView(hFov, aspectRatio) {
+  // Units of the angles: radians.
+  // reference: WbViewpoint::updateFieldOfViewY()
+  var tanHalfFieldOfViewY = Math.tan(0.5 * hFov);
+  // According to VRML standards, the meaning of mFieldOfView depends on the aspect ratio:
+  // the view angle is taken with respect to the largest dimension
+  if (aspectRatio < 1.0)
+    return hFov;
+  else {
```

Comment:
Minor notes: 
- in webots.js, if aspectRatio < 1.0 there is no need to compute tanHalfFieldOfViewY
- this `else` is not needed because of the previous `return`

Reply:
I just simplified it a lot ^^